### DISCUSSION TITLE
Get CD-ROM image question

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#959] [Q&A] [answered]
Does clicking "Get CD-ROM image" just give you a copy of the original uploaded file, or does it upload the changed file if you downloaded any files in the VM?

**Comment by copy** [comment, marked as answer]
If any files were written to disk, you get an updated disk image with those files (you may need to run `sync` or similar). However, since CD-ROMs are always read-only, so this only works with floppy disks or hard disks.
